### PR TITLE
[User] Possibility to clear profile picture

### LINKED ISSUE #6329: [Improvement] Remove user image
# Improvement
## Improvement description
A few days ago i accidentally maintained a picture on the wrong user in a project, then i had to delete the picture on the filesystem, which is very cumbersome.

### PR DESCRIPTION
**Pull request by skochTC**
Resolves #6329 

I also updated this panel and fixed the bug:
grafik

to this (does not look so scary 😉):
grafik

Linked @skoch98 for fixes and questions

### CODE REVIEW

**Comment by brusch** [comment]
@skochTC Can you please resolve the conflicts? 
Thanks a lot!

**Comment by skochTC** [comment]
> @skochTC Can you please resolve the conflicts?
> Thanks a lot!

Sure, I can do it tomorrow 👍

**Comment by skoch98** [comment]
> > @skochTC Can you please resolve the conflicts?
> > Thanks a lot!
> 
> Sure, I can do it tomorrow 👍

Done :-)

File: bundles/AdminBundle/Resources/public/js/pimcore/helpers.js [hunk 1]
```diff
@@ -3180,3 +3180,19 @@ pimcore.helpers.getProgressWindowListeners = function () {
         }
     };
 };
+
+pimcore.helpers.reloadUserImage = function (userId) {
+    var image = "/admin/user/get-image?id=" + userId + "&_dc=" + Ext.Date.now();
```
Comment (blankse) [inline bundles/AdminBundle/Resources/public/js/pimcore/helpers.js:3185]: Should also use JS Routing
```suggestion
    var image = Routing.generate('pimcore_admin_user_getimage', {id: userId, '_dc': Ext.Date.now()});
```

Comment (skoch98) [reply bundles/AdminBundle/Resources/public/js/pimcore/helpers.js]: Thanks

**Comment by dvesh3** [comment]
@skoch98 ncie work 👏 can you please also add clear button to user panel so that admin can also clear the picture and views are consistent ? thanks!
image

**Comment by skoch98** [comment]
> @skoch98 nice work 👏 can you please also add clear button to user panel so that admin can also clear the picture and views are consistent ? thanks!
> image

Hi, sorry for my late reply I am currently on holidays.

I think I've already implemented that?
In this area:
image

Looks like this:
image

On my own profile:
image

**Comment by dvesh3** [comment]
yes, I see it now. Sorry! might be caching issue at my end. enjoy your holidays 😉

**Comment by dvesh3** [comment]
thanks a lot!

**Pull request merged**